src/interface.jl

```diff
@@ -465,7 +467,7 @@ Runs the post solve both for a cleaner solutiona and to optimize
 for the continuous variables if present.
 Prints solution statistics if verbose is true.        
 """
-function postsolve(tree, result, time_ref, verbose, use_postsolve)
+function postsolve(tree, result, time_ref, verbose, use_postsolve, max_iteration_post)
```

Comment:
update the docstring above too

Reply:
I added `max_iteration_post    - maximum number of iterations in a FrankWolfe run during postsolve` in l.26, or do you mean something else?